packages/app-bridge/src/types/GuidelineSearchResult.ts

```diff
@@ -0,0 +1,21 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import type { CamelCasedPropertiesDeep } from 'type-fest';
+
+export type GuidelineSearchResultApi = {
+    highlights: string[];
+    type: 'PAGE' | 'SECTION' | 'BLOCK' | 'COLOR';
+    object_id: number;
+    page_id: number;
+    page_slug: string;
+    page_title: string;
+    page_category_slug: string | null;
+    block_id: number;
+    document_id: number;
+    document_slug: string;
+    document_title: string;
+    portal_id: number;
+    portal_token: string;
```

Comment:
This can also be null:

```suggestion
    portal_token: string | null;
```